doc/source/user_guide/advanced.rst

```diff
@@ -783,10 +783,13 @@ values **not** in the categories, similarly to how you can reindex **any** panda
 
 .. ipython:: python
 
-   df2.reindex(['a', 'e'])
-   df2.reindex(['a', 'e']).index
-   df2.reindex(pd.Categorical(['a', 'e'], categories=list('abcde')))
-   df2.reindex(pd.Categorical(['a', 'e'], categories=list('abcde'))).index
+   df3 = pd.DataFrame({'A': np.arange(3),
+                       'B': pd.Series(list('abc')).astype('category')})
+   df3 = df3.set_index('B')
+   df3.reindex(['a', 'e'])
```

Comment:
you need to show df3, otherwise its not obvious what you are doing. split this into 2 ipython:: python blocks

Reply:
I've tried to clean this up a bit, perhaps it could still be improved though